tbdex/rfq/rfq_test.go

```diff
@@ -439,3 +439,32 @@ func TestRFQ_Verify_FailsBadSignature(t *testing.T) {
 	err = rfq.Verify(false)
 	assert.Error(t, err)
 }
+
+
+func TestRFQ_Verify_InvalidSignature(t *testing.T) {
```

Comment:
Fixing the naming conventions for this in another PR:

```suggestion
func TestVerify_InvalidSignature(t *testing.T) {
```